src/ts/app-config/default-app-config.ts

```diff
@@ -1,11 +1,12 @@
 import { AppConfig } from "./app-config";
 import { join } from "path";
-import { homedir } from "os";
+// import { homedir } from "os";
+import { cwd } from "process";
 
 export class DefaultAppConfigManager {
     public static getDefaultAppConfig(): AppConfig {
         return {
-            userSettingsFilePath: join(homedir(), "ueli.config.json"),
+            userSettingsFilePath: join(cwd(), "ueli.config.json"),
```

Comment:
I want to keep the settings file in the user home directory by default so every user can have his own configuration. In addition your config would change if you decide to install ueli in a different folder afterwards.